src/main/java/ru/sbtqa/tag/pagefactory/drivers/TagWebDriver.java

```diff
@@ -137,11 +138,71 @@ private static void createDriver() throws UnsupportedBrowserException, Malformed
         webDriver.get(WEBDRIVER_STARTING_URL);
     }
 
+    private static void configureDriver(BrowserManager webDriverManager, String browserType) {
+        if (!WEBDRIVER_PATH.isEmpty()) {
+            System.setProperty("webdriver." + browserType + ".driver", new File(WEBDRIVER_PATH).getAbsolutePath());
+        } else {
+            LOG.warn("The value of property 'webdriver.drivers.path is not specified."
+                    + " Trying to automatically download and setup driver.");
+            configureWebDriverManagerParams(webDriverManager, browserType);
+            webDriverManager.setup();
+        }
+    }
+
+    private static void configureProxy(DesiredCapabilities capabilities) {
+        if (!WEBDRIVER_PROXY.isEmpty()) {
+            setProxy(new BrowserMobProxyServer());
+            proxy.start(0);
+            Proxy seleniumProxy = ClientUtil.createSeleniumProxy(proxy);
+            capabilities.setCapability(CapabilityType.PROXY, seleniumProxy);
+        }
+    }
+
+    private static void configureWebDriverManagerParams(BrowserManager webDriverManager, String browserType) {
+        if (!WEBDRIVER_BROWSER_VERSION.isEmpty() && !browserType.equalsIgnoreCase(BrowserType.IE)) {
+            LOG.info("You have specified 'webdriver.browser.version' property. Trying to find corresponding driver.");
+
+            String mappedVersion = parseDriverVersionFromMapping(WEBDRIVER_BROWSER_VERSION, browserType.toLowerCase());
+            webDriverManager.version(mappedVersion);
+        } else if (!WEBDRIVER_DESIRABLE_VERSION.isEmpty()) {
+            webDriverManager.version(WEBDRIVER_DESIRABLE_VERSION);
+        }
+        if (!WEBDRIVER_NEXUS_LINK.isEmpty()) {
+            webDriverManager.useNexus(WEBDRIVER_NEXUS_LINK);
+        }
+    }
+
+    private static String parseDriverVersionFromMapping(String browserVersion, String browserType) {
+        JsonObject mapping = getResourceJsonFileAsJsonObject(MAPPING_FILES_PATH + browserType + MAPPING_FILES_EXTENSION);
+        try {
+            if (null != mapping) {
```

Comment:
what about 
if (null != mapping && null != mapping.get(browserVersion)) {
    return mapping.get(browserVersion).getAsString();
} else {
    LOG.warn("Can't get corresponding driver for {} browser version. Using LATEST driver version.", browserVersion);
}

And you can kick try-catch block out